SmartDeviceLink/SDLLockScreenViewController.m

```diff
@@ -70,6 +70,28 @@ - (void)setBackgroundColor:(UIColor *_Nullable)backgroundColor {
     [self sdl_layoutViews];
 }
 
+- (void)setLockedLabelText:(NSString *_Nullable)lockedLabelText {
+    _lockedLabelText = lockedLabelText;
+    
+    [self sdl_layoutViews];
+}
+
+#pragma mark - Swipe Gesture
+
+- (void)addDismissGestureWithCallback:(SwipeGestureCallbackBlock)swipeGestureCallback {
+    self.dismissGestureCallback = swipeGestureCallback;
+    UISwipeGestureRecognizer *swipeGesture = [[UISwipeGestureRecognizer alloc] initWithTarget:self action:@selector(sdl_didSwipeToDismiss:)];
```

Comment:
Is it possible for this to be called multiple times and multiple dismiss gestures to be added? It seems like it is. This would probably be fixed by holding a reference to the gesture recognizer as a private property, and then checking if it already exists and not re-adding it?

Reply:
Sounds good, pushed changes